src/reducers/systemIntakeReducer.ts

```diff
@@ -1,20 +1,24 @@
-import { STORE_SYSTEM_INTAKE } from '../constants/actions';
-import { SystemIntakeState } from '../types/systemIntake';
+import { SystemIntakeState } from 'types/systemIntake';
 import {
   initialSystemIntakeForm,
   prepareSystemIntakeForApp
-} from '../data/systemIntake';
+} from 'data/systemIntake';
+import { fetchSystemIntake } from 'types/routines';
+import { Action } from 'redux-actions';
```

Comment:
I don't think I understand why we are static typing our "action" object (type, payload) with `redux-actions`. We aren't using `redux-actions` in our app.

As a base level type, can't we use: 
```
type: string;
payload: any;
```

We can get super strict (like in https://redux.js.org/recipes/usage-with-typescript#type-checking-actions--action-creators), but that's probably not the right call now.

Reply:
The reason I used Action from redux-actions is that's the type that is expected from createRoutine. We can make our own action type but that might have some functionality we want later.

Reply:
Oh hmm okay, I see `"@types/redux-actions" "*"` as a dependency of `redux-saga-routines`